Intersect.Client.Framework/Entities/ICritter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Intersect.Client.Framework.Maps;
+using System;
+using System.Collections.Generic;
+
+namespace Intersect.Client.Framework.Entities
+{
+    public interface ICritter
+    {
+        Guid CurrentMap { get; set; }
```

Comment:
`CurrentMapId` or `MapId`, `CurrentMap` is misleading when it's not a reference to the actual map.